spoon-bytecode/pom.xml

```diff
@@ -0,0 +1,545 @@
+<!--
+
+    Copyright (C) 2006-2017 INRIA and contributors
+    Spoon - http://spoon.gforge.inria.fr/
+
+    This software is governed by the CeCILL-C License under French law and
+    abiding by the rules of distribution of free software. You can use, modify
+    and/or redistribute the software under the terms of the CeCILL-C license as
+    circulated by CEA, CNRS and INRIA at http://www.cecill.info.
+
+    This program is distributed in the hope that it will be useful, but WITHOUT
+    ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+    FITNESS FOR A PARTICULAR PURPOSE. See the CeCILL-C License for more details.
+
+    The fact that you are presently reading this means that you have had
+    knowledge of the CeCILL-C license and that you accept its terms.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!--<parent>
+        <groupId>fr.inria.gforge.spoon</groupId>
+        <artifactId>spoon-core</artifactId>
+        <version>7.2.0-SNAPSHOT</version>
+    </parent>-->
+
+    <groupId>fr.inria.gforge.spoon</groupId>
+    <artifactId>spoon-bytecode</artifactId>
```

Comment:
I'm not sure about this name: spoon-decompiler, maybe?

Reply:
Sure, why not. (In my mind, it was a version of spoon the allowed to work on bytecode. Do you think, the "how it is done" is more important to the user?)

Reply:
It was really a way to open the discussion, I really don't know what's better :)

Reply:
How about `spoon-full`?

Reply:
Let's keep spoon-bytecode for now.

Reply:
> spoon-full

no please.

> spoon-decompiler or spoon-bytecode

I like `spoon-decompiler` more, because it express that feature better. Spoon can decompile class (bytecode) and generate AST for that.
But `bytecode` is more general term, which might mean also bytecode generation (= compiling), which is already possible with spoon thanks to included JDK compiler.

Let's wait for feedback of others - if any - before you start to rename it. I can live with both names ;-)

Reply:
+1 for spoon-decompiler then. Thanks @pvojtechovsky for managing to say what was bothering me :)